### PR TITLE
Add link from item to detail page 

### PR DESCRIPTION
**Pull request by Heathercoraje**
relates #34 

When a user clicks an item on list, it takes her to a detail page where it shows in-depth info.

### CODE REVIEW

File: src/containers/detail.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class Details extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+
+  componentWillMount() {
+    // console.log(this.props);
```
Comment (matthewdking) [inline src/containers/detail.jsx]: This comment can be removed

File: src/components/career_detail.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import NavBar from './nav_bar.jsx';
+
+class CareerDetail extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {};
+  }
+  render() {
+    console.log(this.props);
```
Comment (matthewdking) [inline src/components/career_detail.jsx]: Remove console log

File: src/components/career_detail.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+      <div>
+        <NavBar />
+        <h1>{ this.props.selected.title }</h1>
```
Comment (veredrec) [inline src/components/career_detail.jsx]: Do we want all of the data to be in `h1`? :confused:

File: src/components/career_detail.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <h1>{ this.props.selected.image }</h1>
```
Comment (veredrec) [inline src/components/career_detail.jsx]: image in `` :stuck_out_tongue_winking_eye:

**Pull request merged**